src/components/Form/Form.js

```diff
@@ -48,18 +52,69 @@ const Form = ({ userId, showmodal, setShowModal }) => {
         <section className='form-background'>
           <section 
           className='form-wrapper'>
+            {confetti &&
+              <Confetti
+              width={560}
+              height={445} 
+              />
+            }
             <img src={otters} alt='otters' className='form-img'></img>
             <div className='form-content'>
               <h1 className='form-title'>New Friend Form</h1>
               <label className='form-label'>Name:</label>
-              <input type='text' className='form-name' placeholder='Full Name' onChange={handleChange} ref={inputName}></input>
-              <label className='birthday-label'>Birthday: (don't worry if you don't know the year, just pick the month and day)</label>
-              <input type='date' className='form-date' onChange={handleChange} ref={inputDate}></input>
-              <label className='form-label'>Notes:</label>
-              <input type='text' className='form-memo' onChange={handleChange} ref={inputMemo}></input>
-              {error && 
-              <p>Something went wrong, please try again.</p>}
-              <button className='form-button' onClick={handleSubmit}>Submit</button>
+              <input 
+                type='text' 
+                name='name' 
+                className='form-name' 
+                placeholder='Full Name' 
+                onChange={handleChange}>
+              </input>
+              <p 
+                className='error name-error' 
+                style={{visibility: error ? 
+                  "visible" 
+                  : "hidden"}}>
+                *Name required
+              </p>
+              <label 
+                className='birthday-label'>
+                Birthday: (don't worry if you don't know the year, just pick the month and day)
+              </label>
+              <input 
+                type='date' 
+                name='birthday' 
+                className='form-date' 
+                onChange={handleChange}>
+              </input>
+              <p 
+                className='error birthday-error' 
+                style={{visibility: error ? 
+                  "visible" 
+                  : "hidden"}}>
```

Comment:
I like that you made these hide and unhide to the form doesn't shift when they show.